Project-Descriptions-and-Plans/CVA6/CVA6-main-plan-approved-2024+.md

```diff
@@ -0,0 +1,524 @@
+Prior to presenting the gate to the TWG, it is recommended to review the sub-project and PA gate document in a CVA6 meeting.
+
+At the time of writing, the envisioned sub-projects are:
+- CV32A60AX<sup>3</sup>, a 32-bit application core, led by 10xEngineers
+- CV32A65X, a 32-bit embedded core with dual issue<sup>4</sup>, led by TSS
+- CV64A60AV, a 64-bit application core with an interface to a vector coprocessor, led by Axelera
+- a 32-bit real-time core with safety features, led by a TRISTAN project partner<sup>5</sup>
+- a 64-bit application core with safety features, led by a TRISTAN project partner<sup>5</sup>
+
+Notes:<br>
+<sup>3</sup> CV32A60AX was previously named CV32A60X or step1/step2 (in the 2023-06-26 PA gate).<br>
+<sup>4</sup> The dual issue will likely be selected by TSS. If the single issue is finally selected, the part number will become CV32A60X.<br>
+<sup>5</sup> This partner considers joining the OpenHW Group.
+
+Refer to OpenHW [Dashboard](https://github.com/openhwgroup/programs/tree/master/dashboard) for the current list of sub-projects.
+
+### Sister projects
+
+The sister projects are led independently. They synchronize with the CVA6 main and sub-projects as deemed necessary.
+
+At the time of writing, the sister projects are:
+- [CVA6-H](https://github.com/openhwgroup/programs/tree/master/Project-Descriptions-and-Plans/CORE-V-CVA6-H) (CVA6 Hypervisor Extension Support)
+- [CORE-V L1 DCACHE](https://github.com/openhwgroup/programs/tree/master/Project-Descriptions-and-Plans/CORE-V-L1-DCACHE) (High Performance Data Cache)
+- [CORE-V-VEC](https://github.com/openhwgroup/programs/tree/master/Project-Descriptions-and-Plans/CORE-V-VEC-Research)  (vector coprocessor)
+- [CORE-V CV-Mesh](https://github.com/openhwgroup/programs/tree/master/Project-Descriptions-and-Plans/CV-MESH) (multi-core coherence system)
+- [Tightly-coupled cache coherence for CVA6](https://github.com/openhwgroup/programs/tree/master/Project-Descriptions-and-Plans/CVA6_tightly_coupled_cache_coherence)
+- [CVA6 Free-RTOS](https://github.com/openhwgroup/programs/blob/master/Project-Descriptions-and-Plans/FreeRTOS/PA%20for%20CVA6%20FreeRTOS%20support.md)
+- [Superscalar CVA6](https://github.com/openhwgroup/programs/tree/master/Project-Descriptions-and-Plans/cva6-dual-issue)
+
+## Project Leader(s)
+
+For the main project:
+- Project Manager (PM) and Technical project leader (TPL): Jérôme Quévremont, TRT
+- Verification leader and main committer: Jean-Roch Coulon, TSS
+- FPGA softcore leader: Sébastien Jacq, TRT
+
+The sub-projects will name their leadership teams.
+
+## Project Planning Documents
+
+The main project workplan is below.
+
+## Summary of requirements
+
+### CVA6 features
+
+The CVA6 specification was prepared for the 2022-02-28 PA gate and approved by the Technical WG. It is available in [ReadTheDocs](https://docs.openhwgroup.org/projects/cva6-user-manual/02_cva6_requirements/cva6_requirements_specification.html).
```

Comment:
Because of the ongoing "parameterization", there is some inherent instability in these parameter lists.  This is known to the project. I do not see what I should include in this PA document.